scale/job/models.py

```diff
@@ -2460,13 +2465,16 @@ def create_job_type_v6(self, manifest_dict, trigger_rule_dict=None, configuratio
                 raise Exception('%s is not an editable field' % field_name)
         self._validate_job_type_fields(**kwargs)
 
-        trigger_rule = self._create_seed_job_trigger_rule(manifest, trigger_rule_dict)
+        trigger_rule = None
+        if trigger_rule_dict:
+            trigger_rule = self._create_seed_job_trigger_rule(manifest, trigger_rule_dict)
```

Comment:
Go ahead and remove trigger_rule_dict from this method. v6 will not support trigger rules.

Reply:
Also remove `**kwargs`. We should only allow specific parameters into this method and set specific fields on the job type model.

Reply:
Also remove `_create_seed_job_trigger_rule()`

Reply:
Also make the same changes to `edit_job_type_v6()`. For editing, remove trigger rule and kwargs, and make sure editable fields like is_active are passed in. Also pass in the configuration object just like `create_job_type_v6()`. Pass in the manifest class as well instead of the dict and make sure the manifest updates fields like title, description etc when editing.

Reply:
How does `create_job_type_v6()` work with existing job types? It seems like it always saves a new one which would error when an existing job type with the same name and version already exists.

EDIT: Never mind, I see you split the logic in the view. That's fine.